examples/api-samples/README.md

```diff
@@ -0,0 +1,9 @@
+# Theia - Examples - API Samples - Label Provider
```

Comment:
We should add a new extension to top-level tsconfig.json. Otherwise reference search does not work for it. Try to find who implement `LabelProviderContribution.didChange`